Assets/Scripts/UI/EqualSpacingCalculator.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Netherlands3D.Twin
+{
+    [RequireComponent(typeof(GridLayoutGroup))]
+    public class EqualSpacingCalculator : MonoBehaviour
```

Comment:
Deze is leuk om te delen met het team; dat iedereen weet dat die er is. Dat kan best handig zijn bij het bouwen van UIs!